embed/src/form/form.ts

```diff
@@ -12,6 +12,13 @@ class HyvorPostForm extends HTMLElement {
 		const style = document.createElement('style');
 		style.textContent = formCss;
 		this.shadowRoot!.appendChild(style);
+
+		window.addEventListener(
```

Comment:
@Nadil-K what do you think?

On the one hand, I think we should not integrate external systems directly within the embed.

On the other hand, Hyvor Blogs is our own platform. It generally makes sense to have something like this in HP.

Initially, my thought was to handle this at the installation time, by giving the user a code snippet with a code like this included